test/js/widgets/SelectWidget.unit.spec.jsx

```diff
@@ -20,7 +20,7 @@ describe('Schemaform <SelectWidget>', () => {
         id="testing"
         onChange={onChange}
         options={{ enumOptions }}/>
-    );
+    ).dive(['SelectWidget']);
```

Comment:
Okay, so this is needed because `shallow` only renders one level and [SelectWidget exports a wrapped component](https://github.com/usds/us-forms-system/blob/a963c57f1ebcb39ba6c4c05d5ad6909d676f2198/src/js/widgets/SelectWidget.jsx#L58-L62) unlike the other widgets. So this goes down a level and renders the actual raw widget.

Reply:
Yeah, I'm just not sure why the recompose change is forcing us to go a level down.

Reply:
It doesn't look like it unwraps automatically anymore since https://github.com/acdlite/recompose/pull/473 ? And since their versioning hasn't gotten to 1.0.0 they don't have to worry about breaking changes in minor/patch releases. Yay.